doc/man3/CRYPTO_THREAD_run_once.pod

```diff
@@ -98,6 +127,43 @@ read by CRYPTO_atomic_load() then CRYPTO_atomic_load() must be the only way that
 the variable is read. If atomic operations are not supported and I<lock> is
 NULL, then the function will fail.
 
+=item *
+
+CRYPTO_MUTEX_create() creates an architecture dependent mutex object.
+
+=item *
+
+CRYPTO_MUTEX_init() initializes a mutex object from CRYPTO_MUTEX_CREATE().
+CRYPTO_MUTEX_destroy() deinitializes the mutex and frees associated data
+
+=item *
+
+CRYPTO_MUTEX_lock() and CRYPTO_MUTEX_unlock() locks and unlocks the mutex,
+respectively.
+
+=item *
+
+CRYPTO_CONDVAR_create() creates a conditional variable. CRYPTO_CONDVAR_destroy()
+deinitializes frees associated data with the conditional variable
+
+=item *
+
+CRYPTO_CONDVAR_init() initializes a conditional variable.
+CRYPTO_CONDVAR_wait() expects an acquired mutex and awaits a broadcast.
+CRYPTO_CONDVAR_broadcast() executes a broadcast for a conditional variable.
+
+=item *
+
+CRYPTO_THREAD_enable() enables OpenSSL to spawn threads for a context.
+To impose an upper bound on the number of threads via I<max_threads>,
+or use -1 to impose no limit. CRYPTO_THREAD_cap() can be used to change
+the limit. To disable threads, use CRYPTO_THREAD_disable().
+CRYPTO_THREAD_enabled() indicates whether threading is enabled or not.
+
+CRYPTO_THREAD_spawn_worker() spawns a worker for use by OpenSSL. A callback
+I<cb> may be provided to to CRYPTO_THREAD_spawn_worker() to control the
+lifespan of a worker.
+
 =back
 
 =head1 RETURN VALUES
```

Comment:
It seems the "RETURN VALUES" section needs a bit more filling in.

Reply:
All of the newly added
```
 int CRYPTO_THREAD_cap(OSSL_LIB_CTX *ctx, int max_threads);
 int CRYPTO_THREAD_disable(OSSL_LIB_CTX *ctx);
 int CRYPTO_THREAD_enable(OSSL_LIB_CTX *ctx, int max_threads);
 int CRYPTO_THREAD_enabled(OSSL_LIB_CTX *ctx);
 int CRYPTO_THREAD_spawn_worker(OSSL_LIB_CTX *ctx, CRYPTO_WORKER_CALLBACK cb);
```
except for `CRYPTO_mem_barrier()` that returns nothing, follow the added
> The other functions return 1 on success, or 0 on error.

remark. What amount of detail would you prefer? Should I perhaps list them explicitly, or include a possible overview of error causes?